internal/provider/resource_aci_rest_managed_test.go

```diff
@@ -331,12 +313,90 @@ func TestAccAciRestManaged_import(t *testing.T) {
+				ResourceName:  "aci_rest_managed.eptags_import_json_str",
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "content.annotation", "orchestrator:from_resource"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "dn", "uni/tn-"+name+"/eptags"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.#", "4"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "annotation", "orchestrator:from_resource"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "class_name", "fvEpTags"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.0.class_name", "tagAnnotation"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.0.rn", "annotationKey-[~!$([])_+-={};:|,.]"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.0.content.key", "~!$([])_+-={};:|,."),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.0.content.value", "tagAnnotation1"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.1.class_name", "tagAnnotation"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.1.rn", "annotationKey-[tagAnnotation2]"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.1.content.key", "tagAnnotation2"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.1.content.value", "tagAnnotation2"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.2.class_name", "fvEpMacTag"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.2.rn", "epmactag-90:B5:B8:42:D1:88-[default]"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.2.content.bdName", "default"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.2.content.mac", "90:B5:B8:42:D1:88"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.3.class_name", "fvEpIpTag"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.3.rn", "epiptag-[2001:10:1::1]-default"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.3.content.ip", "2001:10:1::1"),
+					resource.TestCheckResourceAttr("aci_rest_managed.eptags_import_json_str", "child.3.content.bdName", "default"),
+				),
+			},
+			{
+
+				ImportState:   true,
+				ImportStateId: fmt.Sprintf(`{ "parentDn": "uni/tn-%s/BD-bd_test", "childRns": [ "rgexpmac-00:22:BD:F8:11:FF", "rgexpmac-00:22:BD:F8:12:FF" ] }`, name),
```

Comment:
Is there a test case for the simple uni/tn-{name} case without children and without JSON?

Reply:
Yes, we have it.